compatibility.bs

```diff
@@ -1055,11 +1057,58 @@ On mobile platforms, including smaller iPad form factors
   </tbody>
 </table>
 
+
+<h2 id="removal">Deprecation and Removal</h2>
+
+<em>This section is non-normative.</em>
+
+While the web platform stives for backwards-compatibility, there are occasions which require the
+outright removal or discouraged use of some features. To support making these changes
```

Comment:
The table only seems to allow for removal, not discouragement. Should we document synchronous XMLHttpRequest?

Reply:
I'd consider an entry with `Removal` empty, but `Deprecated from` not empty to be discouraged.

A definition would be useful to clarify that